### PR TITLE
Live update 04/2023

### PR DESCRIPTION
**Pull request by kpodemski**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

File: content/news/public-oss-demo/2023/live-update-4-2023.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+layout: post
+title:  "PrestaShop Live Update - April 2023"
+subtitle: "A closer look at what happened in the project last month"
+date:   2023-04-13
+authors: [ kpodemski ]
+icon: icon-video-camera
+tags: [contribute, event, community, live-update]
+---
+
+The project's live update was held for the fourth time this year and [streamed live](hhttps://www.youtube.com/watch?v=SVbAQ__A6ns) on March 8th, 2023.
```
Comment (coffrep) [inline content/news/public-oss-demo/2023/live-update-4-2023.md]:
```suggestion
The project's live update was held for the fourth time this year and [streamed live](hhttps://www.youtube.com/watch?v=SVbAQ__A6ns) on April 12th, 2023.
```

File: content/news/public-oss-demo/2023/live-update-4-2023.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## The session's agenda was as follows:
+
+- [00:00](hhttps://www.youtube.com/watch?v=SVbAQ__A6ns) Beginning of the session: the introduction
+- [01:54](https://youtu.be/SVbAQ__A6ns?t=114) Project news: PrestaShop 8.0.2 and 8.0.3 releases
+- [03:11](https://youtu.be/SVbAQ__A6ns?t=191) Project news: A new organization for the project
```
Comment (coffrep) [inline content/news/public-oss-demo/2023/live-update-4-2023.md]:
```suggestion
- [03:11](https://youtu.be/SVbAQ__A6ns?t=191) Project news: a new organization for the project
```

File: content/news/public-oss-demo/2023/live-update-4-2023.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- [44:05](https://youtu.be/SVbAQ__A6ns?t=2644) Demo: Developer documentation improvements, by [@kpodemski](https://github.com/kpodemski) and [@thomasnares](https://github.com/thomasnares)
```
Comment (coffrep) [inline content/news/public-oss-demo/2023/live-update-4-2023.md]:
```suggestion
- [44:05](https://youtu.be/SVbAQ__A6ns?t=2644) Demo: developer documentation improvements, by [@kpodemski](https://github.com/kpodemski) and [@thomasnares](https://github.com/thomasnares)
```

**Review by coffrep — CHANGES_REQUESTED** [review]
Hi @kpodemski - I've added some suggestions, including for the date.

File: content/news/public-oss-demo/2023/live-update-4-2023.md (hunk 1 above)
Comment (kpodemski) [reply content/news/public-oss-demo/2023/live-update-4-2023.md]: nice catch

**Review by coffrep — APPROVED** [review]

**Pull request merged**